### PR TITLE
Added Intelligence notifications feed.

### PR DESCRIPTION
**Pull request by keithjjones**
I added the ability to pull the intelligence hunting notification feed.

### CODE REVIEW

File: virus_total_apis/api.py [hunk 1]
```diff
@@ -916,6 +916,26 @@ def get_all_file_report_pages(self, query):
...
+        params = {'apikey': self.api_key, 'page': page}
+        try:
+            response = requests.get(self.base + 'hunting/notifications-feed/',
+                                    params=params,
+                                    proxies=self.proxies,
+                                    timeout=timeout)
+        except requests.RequestException as e:
+            return dict(error=e.message)
+
+        return response.json()['next'], response
```
Comment (MrRedAmber) [inline virus_total_apis/api.py]: Please use `dict.get('next')` instead of accessing the dictionary by `dict['next']`

Comment (MrRedAmber) [inline virus_total_apis/api.py]: Can you please also update the `HISTORY.rst` file accordingly.

Comment (keithjjones) [inline virus_total_apis/api.py]: Sorry, I just saw this.  Did you get it? @MrRedAmber and @blacktop ?

**Comment by blacktop** [comment]
ya we got it in, I will push to PyPi soon.  Thanks for the contribution 👍

**Comment by keithjjones** [comment]
Anytime!  👍

**Pull request merged**